composer.json

```diff
@@ -15,25 +15,33 @@
     },
     "require-dev": {
         "mediact/testing-suite": "@stable",
-        "composer/composer": "^1.3 || ^2.0",
         "kint-php/kint": "@stable",
-        "mediact/coding-standard": "@stable",
-        "phpstan/phpstan": "@stable",
+        "mediact/coding-standard": "dev-feature/OTPBBBBBZP-117",
         "mikey179/vfsstream": "^1.6",
         "ext-simplexml": "*"
     },
+    "repositories": [
```

Comment:
Do we really need these repo's? I guess for the dev-feature requirement perhaps? But otherwise we can just push releases to packagist.